rest_tools/client/device_client.py

```diff
@@ -165,6 +165,8 @@ def SavedDeviceGrantAuth(
     client_id: str,
     client_secret: Optional[str] = None,
     scopes: Optional[List[str]] = None,
+    timeout: float = 60.0,
+    retries: int = 10,
```

Comment:
I don't think we want to define the defaults here, we only want to forward values if the user specifies them.

Example:
```python
def foo(myarg=None):
    kwargs = {}
    if myarg:
        kwargs['myarg'] = myarg
    bar(**kwargs)
```

Reply:
I've retooled them as a kwargs pass-through.
Is that more what you're thinking here?